src/components/Whiteboard/ElementBehaviors/Resizable/ResizeElement.test.tsx

```diff
@@ -157,4 +168,57 @@ describe('<ResizeElement />', () => {
       type: 'path',
     });
   });
+
+  it('should resize line elements', () => {
+    render(<ResizeElement elementId="element-1" />, {
+      wrapper: Wrapper,
+    });
+
+    // drag a resize box from 0,0 to 50,50
+    const resizeHandleBottomRight = screen.getByTestId(
+      'resize-handle-se-resize',
+    );
+    mocked(calculateDimensions).mockReturnValue({
+      x: 0,
+      y: 0,
+      width: 0,
+      height: 0,
+    });
+    fireEvent.mouseDown(resizeHandleBottomRight);
+    mocked(calculateDimensions).mockReturnValue({
+      x: 0,
+      y: 0,
+      width: 50,
+      height: 50,
+      points: [
+        { x: 0, y: 0 },
+        { x: 50, y: 50 },
+      ],
+    });
+    fireEvent.mouseMove(resizeHandleBottomRight);
+    fireEvent.mouseUp(resizeHandleBottomRight);
+
+    const element = activeSlide.getElement('element-1');
+    expect(element).toEqual({
```

Comment:
If the line starts at `0, 0` and ends at `1, 1` and the test drags from `0, 0` to `50, 50` then I would expect the result to be `1, 1`  /  `50, 50`.

Reply:
Nice catch!